client/tsconfig.paths.json

```
@@ -2,7 +2,15 @@
   "compilerOptions": {
     "baseUrl": ".",
     "paths": {
-      "~shared/*": ["../shared/src/*"]
+      "~shared/*": ["../shared/src/*"],
+      "@api/*": ["src/api/*"],
+      "@assets/*": ["src/assets/*"],
+      "@components/*": ["src/components/*"],
+      "@contexts/*": ["src/contexts/*"],
+      "@hooks/*": ["src/hooks/*"],
+      "@services/*": ["src/services/*"],
+      "@theme/*": ["src/theme/*"],
+      "@util/*": ["src/util/*"]
```

okay, this was precisely what I wasn't very happy with. could we perhaps change this to...
```suggestion
      "@/*": ["src/*"]
```

This might mean that we will have to rework all the commits in this PR though, but it avoids bloating tsconfig, and avoids having to remember to add an entry to tsconfig when we start a new subdir in `src/`

---

Will use the latter !